images/node/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM registry.svc.ci.openshift.org/openshift/builder:golang-1.12 AS builder
+WORKDIR /go/src/github.com/openshift/sdn
+COPY . .
+RUN go build "vendor/github.com/containernetworking/plugins/plugins/ipam/host-local"
+RUN go build "vendor/github.com/containernetworking/plugins/plugins/main/loopback"
+RUN mkdir -p /tmp/build; \
+    cp /go/src/github.com/openshift/origin/_output/local/bin/linux/$(go env GOARCH)/openshift-sdn /tmp/build/openshift-sdn; \
```

Comment:
this won't match

Reply:
@sttts this break the build

Reply:
```
COPY --from=builder /go/src/github.com/openshift/sdn/openshift-sdn /usr/bin/
```

Reply:
or pick this file https://github.com/openshift/origin/pull/23273/files#diff-29391d0364ab2ca3cb45701b081dcf3aR1